mmtrack/models/backbones/mixformer_backbone.py

```diff
@@ -89,6 +108,12 @@ def __init__(self,
 
     def _build_projection(self, dim_in, dim_out, kernel_size, padding, stride,
                           method):
+        """Build qkv projection according to `method` argument.
+
+        "dw_bn": Apply a convolution layer with batch norm on the input.
```

Comment:
Please reorganize all docstring format like other methods in `mmtracking`. You can use `autoDocstring`  plugin in vsocde for convenience.
For some tensor variables, it's better to describe their shapes for easy understanding.